crates/cranelift/src/builder.rs

```diff
@@ -31,34 +33,20 @@ pub struct LinkOptions {
 }
 
 pub fn builder() -> Box<dyn CompilerBuilder> {
-    let mut flags = settings::builder();
-
-    // There are two possible traps for division, and this way
-    // we get the proper one if code traps.
-    flags
-        .enable("avoid_div_traps")
-        .expect("should be valid flag");
-
-    // We don't use probestack as a stack limit mechanism
-    flags
-        .set("enable_probestack", "false")
-        .expect("should be valid flag");
-
     Box::new(Builder {
-        flags,
-        isa_flags: cranelift_native::builder().expect("host machine is not a supported target"),
```

Comment:
I see the refactoring in cranelift-native, but I think that the usage of `cranelift_native` here may be lost? I think part of `IsaBuilder::new` may want to pass the configured flags to the native configuration crate as well?

Reply:
Oh yes thanks good catch! This needs to be added back in

Reply:
Ah I see, I didn't realize that the initial version of `isa_flags` is this builder returned from `cranelift_native` but any time you call `target` uses the `lookup` function from the `isa` module 🤔 I'll have to rework this a bit so the caller can pass an initial builder that might differ from the one `lookup` returns.

Reply:
Ah nvm on that, I see your previous change with the `infer_native_flags`. I've added this back in